src/cppcache/integration-test/testThinClientPoolExecuteFunction.cpp

```diff
@@ -1506,48 +1353,10 @@ void runFunctionExecution(bool isEndpoint) {
   CALL_TASK(StopC1);
   CALL_TASK(CloseServers);
   CALL_TASK(CloseLocator1);
-
```

Comment:
Like I said in the review request. This test was split out into tho two tests. The test had two different tests crammed into one file. The big delete in this file is all the crap that became the second test.